css/style.css

```diff
@@ -1,8 +1,122 @@
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- */
+
+ * {
+    box-sizing: border-box;
+  }
+  
+  /* Add a gray background color with some padding */
+  body {
+    font-family: Arial;
+    padding: 20px;
+    background: #eaebee;
+  }
+  
+  /* Header/Blog Title */
+  .header {
+    padding: 30px;
+    font-size: 60px;
+    text-align: center;
+    background: white;
+    color: rgb(112, 80, 80);
+    text-shadow: 2px 2px 8px #272424
+  }
+  .topnav {
+    overflow: hidden;
+    background-color: rgb(92, 81, 81);
+    
+  }
+  
+  .topnav a {
+    display: flex;
+    justify-content: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 27px;
+  }
+  
+  .topnav a:hover {
+    background-color: #ddd;
+  }
+  
+  .topnav a.active1{
+    background-color: #28cc90;
+    color: white;
+    
+  }
+
+  .active2{
+    background-color: #ffffff;
+    color: rgb(179, 32, 32);
+  }
+  .active3{
+    background-color: #cf3345;
+    color: white;
+  }
```

Comment:
I loved how you played with active 1, 2, and 3, so that the colors make the flag of Iran. Such a cool idea!